language/move-lang/src/parser/syntax.rs

```diff
@@ -579,14 +579,16 @@ fn parse_term<'input>(tokens: &mut Lexer<'input>) -> Result<Exp, Error> {
         Tok::NameValue => {
```

Comment:
Please update the grammar comments before parse_term. E.G., combining the two lines starting with \<ModuleAccess\> so that they end with: (("{" Comma\<ExpField\> "}") | ("(" Comma\<Exp\> ")"))?

Or maybe it would be more clear to add a 3rd \<ModuleAccess\> line....

Reply:
Done.